### PR TITLE
Added an option page with a color scheme picker

### LINKED ISSUE #39: Option menu
To access the option for colors.

Related to #38

### PR DESCRIPTION
**Pull request by YonatanNitzan**
Added a display of the color scheme when you pick one
Also made color picker more generic (self generating)

Uses local storage to store scheme number

closes #39

### CODE REVIEW

File: src/views/options-menu.js [hunk 1]
```diff
@@ -0,0 +1,65 @@
+import React, {Component} from 'react'
+import MainMenu from './main-menu'
+import {pickScheme, colorSchemes, getColorScheme} from '../components/charts/chart-builder'
+
+class OptionsMenu extends Component {
+  constructor (props) {
+    super(props)
+    var currentScheme = getColorScheme(localStorage.schemeNum)
```
Comment (TheArchNemesis) [inline src/views/options-menu.js]: make this const

File: src/components/charts/chart-builder.js [hunk 2]
```diff
@@ -1,17 +1,25 @@
...
-  'default': ['rgba(46, 139, 87, 0.8)', 'rgba(220, 20, 60, 0.8)', 'rgba(64, 224, 208, 0.8)', 'rgba(192, 192, 192, 0.8)', 'rgba(255, 218, 185, 0.8)', 'rgba(75, 0, 130, 0.8)'],
-  'redGreen': ['rgba(255, 215, 0, 0.8)', 'rgba(220, 20, 60, 0.8)', 'rgba(173, 200, 255, 0.8)', 'rgba(192, 192, 192, 0.8)', 'rgba(244, 164, 96, 0.8)', 'rgba(0, 0, 128, 0.8)'],
-  'greyScale': ['rgba(0, 0, 0, 0.8)', 'rgba(47, 79, 79, 0.8)', 'rgba(119, 136, 153, 0.8)', 'rgba(128, 128, 128, 0.8)', 'rgba(192, 192, 192, 0.8)', 'rgba(220, 220, 220, 0.8)']
+export const colorSchemes = {
+  'default': {
+    name: 'Normal',
+    colors: ['rgba(46, 139, 87, 0.8)', 'rgba(220, 20, 60, 0.8)', 'rgba(64, 224, 208, 0.8)', 'rgba(192, 192, 192, 0.8)', 'rgba(255, 218, 185, 0.8)', 'rgba(75, 0, 130, 0.8)']
+  },
+  'redGreen': {
+    name: 'Sagi Mode',
```
Comment (TheArchNemesis) [inline src/components/charts/chart-builder.js]: Please find out the name of Sagi's illness and name this color appropriately.

File: src/views/options-menu.js [hunk 3]
```diff
@@ -0,0 +1,65 @@
+import React, {Component} from 'react'
+import MainMenu from './main-menu'
+import {pickScheme, colorSchemes, getColorScheme} from '../components/charts/chart-builder'
+
+class OptionsMenu extends Component {
```
Comment (TheArchNemesis) [inline src/views/options-menu.js]: please rename options to settings

File: src/views/main-menu.js [hunk 4]
```diff
@@ -34,6 +34,17 @@ class MainMenu extends Component {
             width: '25vw'
           }}>Info Hub
           </button>
+          <button className="btn btn-warning" onClick={() => {
```
Comment (TheArchNemesis) [inline src/views/main-menu.js]: please make this button btn-seconday

File: src/components/charts/chart-builder.js (hunk 2 above)
Comment (YonatanNitzan) [reply src/components/charts/chart-builder.js]: I have been specifically requested to call it Sagi Mode

File: src/views/main-menu.js (hunk 4 above)
Comment (YonatanNitzan) [reply src/views/main-menu.js]: But it needs to look different, I think

File: src/views/options-menu.js (hunk 1 above)
Comment (YonatanNitzan) [reply src/views/options-menu.js]: Can't, I update it's value

**Comment by YonatanNitzan** [comment]
closes #39

File: src/components/charts/chart-builder.js (hunk 2 above)
Comment (TheArchNemesis) [reply src/components/charts/chart-builder.js]: I doubt that the one who requested that was serios. please rename it to "Deutranopia" mode. that is the name of Sagi's illness.

Comment (YonatanNitzan) [reply src/components/charts/chart-builder.js]: Done

File: src/views/main-menu.js (hunk 4 above)
Comment (TheArchNemesis) [reply src/views/main-menu.js]: In my opinion, the main menu should be coherent

**Pull request merged**